gpfit/tests/t_initialize.py

```diff
@@ -13,45 +13,58 @@ class TestMaxAffineInitK2(unittest.TestCase):
     This unit test ensures that max affine init produces an array
     of the expected shape and size
     """
-    x = arange(0., 16.)[:, newaxis]
-    y = arange(0., 16.)[:, newaxis]
+
+    x = arange(0.0, 16.0)[:, newaxis]
+    y = arange(0.0, 16.0)[:, newaxis]
     K = 2
-    ba = ba_init(x, y, K)
+    ba = get_initial_parameters(x, y, K)
 
     def test_ba_ndim_k2(self):
         self.assertEqual(self.ba.ndim, 2)
 
     def test_ba_shape_k2(self):
 
         _, dimx = self.x.shape
-        self.assertEqual(self.ba.shape, (dimx+1, self.K))
+        self.assertEqual(self.ba.shape, (dimx + 1, self.K))
 
 
 class TestMaxAffineInitK4(unittest.TestCase):
     """
     This unit test ensures that max affine init produces an array
     of the expected shape and size
     """
+
     np.random.seed(SEED)
-    Vdd = random_sample(1000,) + 1
-    Vth = 0.2*random_sample(1000,) + 0.2
-    P = Vdd**2 + 30*Vdd*exp(-(Vth-0.06*Vdd)/0.039)
+    Vdd = (
+        random_sample(
+            1000,
+        )
+        + 1
+    )
+    Vth = (
+        0.2
+        * random_sample(
+            1000,
+        )
+        + 0.2
+    )
```

Comment:
hmmmm

Reply:
another `black` special but will fix this by taking away the `,`